### PR TITLE
relationship updated

### PR DESCRIPTION
**Pull request by livlaming**
ola!
short and little and

### CODE REVIEW

**Review by Ryno95 — COMMENTED** [review]
Looks good in general , one class validator thing.

Question about roles, how do you add a role to a chat-user?

File: backend/src/chats/chat/entities/chat.entity.ts [hunk 1]
```diff
@@ -29,6 +31,6 @@ export class Chat {
 	@Exclude()
 	password: string;
```
[the remark this answers is not in the document]
Comment (vincentvis) [reply backend/src/chats/chat/entities/chat.entity.ts:32]: Changing it to `@Column({ nullable: true })` solves this, right?

[the remark this answers is not in the document]
Comment (livlaming) [reply backend/src/chats/chat/entities/chat.entity.ts:32]: I'll try this out :)

Ps @Ryno95  the above question about the roles, (I cant reply for some reason) so here we go, there is a userchatrole class.

File: backend/src/chats/chat/entities/chat.entity.ts [hunk 2]
```diff
@@ -1,12 +1,14 @@
 import {
 	Column,
 	Entity,
+	JoinColumn,
```
Comment (vincentvis) [inline backend/src/chats/chat/entities/chat.entity.ts:4]: Added but not used

File: backend/test/chat.e2e-spec.ts [hunk 3]
```diff
@@ -5,10 +5,21 @@ import { ChatController } from '../src/chats/chat/chat.controller';
...
 	let app: INestApplication;
 	let chatController: ChatController;
+	let messageService: MessageService;
+	let messageController: MessageController;
+
+	const testMessages: CreateMessageDto[] = [
+		{ sender_id: 1, chat_id: 1, content: 'hello' },
+		{ sender_id: 2, chat_id: 1, content: 'world!' },
+	];
+
```
Comment (vincentvis) [inline backend/test/chat.e2e-spec.ts:22]: unused (also MockChat is unused)
also mentioned in other review

**Review by vincentvis — APPROVED** [review]

**Review by vincentvis — APPROVED** [review]

**Review by Ryno95 — APPROVED** [review]
ShapShapMieliePap!

**Pull request merged**